src/main/java/org/jsoup/internal/StringUtil.java

```diff
@@ -5,11 +5,14 @@
 
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Iterator;
```

Comment:
Can you change your format settings so that we don't use wildcard imports? Prefer to list them all explicitly.